README.md

```diff
@@ -18,11 +18,11 @@ In order to create an Ogma visualization with your Linkurious styles:
 
 1. Get your visualization using [Linkurios REST API](https://doc.linkurio.us/server-sdk/latest/apidoc/#api-Visualization-getVisualization).
```

Comment:
`Linkurios REST API` there is a type and we should use the full name: `Linkurious Enterprise REST API` (there are a couple of other instances where wither there is a typo or we just use `Linkurious` instead of `Linkurious Enterprise`